dask/dataframe/core.py

```diff
@@ -336,6 +335,25 @@ def quantiles(self, q):
         """
         return quantiles(self, q)
 
+    def resample(self, rule, how=None, axis=0, fill_method=None, closed=None,
+                 label=None, convention='start', kind=None, loffset=None,
+                 limit=None, base=0):
+        first, last = self.divisions[0], self.divisions[-1]
+        new_divisions = list(pd.date_range(start=first, end=last, freq=rule))
```

Comment:
> Would it be possible to sync up each new division to correspond with the rough location of the old one but aligned on the new grid?

So, just to make sure I understand what you mean by this:

Let's say I have data split into

```
January, February, October, November
```

and i want to resample minute by minute, then the new_divisions should be

```
1st minute of january, 1st minute of february, 1st minute of october, 1st minute of november
```
?